### ISSUE TITLE
[VueJS] : Dynamically set initial value of editor

### ISSUE THREAD

**Issue by ezawadzki** [nhn/tui.editor#1728]
Hi,

I'm trying to init value of my editor on project using vuejs. Just want to replace "init value" by "another value" dynamically, in the aim to call an API. I just still get "init value". Tried to reuse code from example or issues, but something not working.

Here is my editor:

```
<editor
  ref="toastuiEditor"
  :initialValue="editorText"
  initialEditType="markdown"
  @load="onEditorLoad"
  @change="onEditorChange"
  @stateChange="onEditorStateChange"
/>
```
About data :

```
data() {
  return {
      editorText: "init value",
  }
}
```

And in script part, I'm trying one of these solution, no one work :

```
onEditorLoad() {
  // Solution 1
  this.editorText = "another text";

  // Solution 2
  this.$refs.toastuiEditor.initialValue = "another value";

  // Solution 3
  this.$refs.toastuiEditor.invoke("setMarkdown", "another value", true);

  // Solution 4
  this.$refs.toastuiEditor.invoke("setHTML", "another value", true);

}
```

Versions :
```
"vue": "2.6.11",
"@toast-ui/vue-editor": "^2.5.3",
```

Thanks for this great job, this editor seems to be awesome !

**Comment by ats1999** [comment]
You can use the [API](https://nhn.github.io/tui.editor/latest/ToastUIEditorCore#setMarkdown)

Here is how i used in React, `ref.current.getInstance().setMarkdown("# rahul\n")` . You need to first get the editor instance as per vue docs, then call the API.

**Comment by ezawadzki** [comment]
Hi @ats1999,

Thanks for your answer, but this is exactly what I do but with vueJS.
It's seems to be resolved in last version `3.0.2`.
